exporter/metric/stdout/stdout.go

```diff
@@ -98,6 +101,20 @@ func New(options Options) (*Exporter, error) {
 	}, nil
 }
 
+// NewExportPipeline sets up a complete export pipeline with the recommended setup
```

Comment:
Looks good. Will you add a comment here explaining the standard usage is something like

```
   metricSDK, err := stdout.NewExportPipeline(stdout.Options{...})
   if err != nil {
       ...
    }
   defer metricSDK.Stop()

   global.SetMeterProvider(metricSDK)

   // ... Done!
```

Reply:
Considering that some exporters should be created with specific `selectors`, `batchers` and so on, should we standardize that only the `NewExportPipeline` is exported on all supported exporters? (and maybe rename it to `New()`?) 

Is there an use for exposing the `stdout.New()` with this new additions? Or should all customization be done through options? 

I'm asking this because users could misuse exporters by only creating an `stdout.New()`, for example, and thinking that this is enough to have an exporter configured. Or even creating invalid configurations.

Reply:
Maybe we should avoid `New` altogether and export two functions: `NewPipeline()` to build and start a pipeline, `NewExporter()` to return a plain exporter that could be explicitly configured.

Reply:
can we also set the provider in NewPipeline(). So it would look
```go
pipeline, err := stdout.NewPipeline(stdout.Options{...})
if err != nil {
 ...
}
defer pipeline.Stop()
// ... Done
```

Reply:
I would probably rename the method if it also sets the global provider. Maybe `Install()`

Reply:
Pushed commit that addresses this.